src/Dotnet.Script.Core/Templates/program.publish.template

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Scripting.Hosting;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+using System.Threading.Tasks;
+using static System.Console;
+
+namespace dotnetPublishCode
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                var globals = new CommandLineScriptGlobals(Console.Out, CSharpObjectFormatter.Instance);
+                foreach (var arg in args)
+                    globals.Args.Add(arg);
+
+                var factoryMethod = typeof(scriptAssembly).GetMethod("<Factory>");
+                if (factoryMethod == null) throw new Exception("couldn't find factory method to initiate script");
+
+                // todo: not sure what the second parameter is, using 'null' for now
```

Comment:
this is OK - the second index in the array is for submission continuations (REPL scenarios)